README.md

```diff
@@ -0,0 +1,40 @@
+# My INF 7429 Project
+
+I am going to build a food blog website. The main intended purpose of the project will be to develop a site that will showcase a variety of Keto recipes that I've tried, and created.
+
+## Wireframes
+
+When thinking about Information Architecture, and the creation of websites, I love creating wireframes to pre-plan the site's layout. Here are a few wireframes for what I have in mind with this site.
+
+![Wireframe of the Landing Page] (wireframes/homepage.jpg)
```

Comment:
There are spaces in between the bracket and parentheses `...Page] (wire...` that are preventing the image of the wireframes from being displayed in the README file. Please make this change. Thank you.